src/pages/trade/HorizontalPanel.tsx

```diff
@@ -84,6 +83,9 @@ export const HorizontalPanel = ({ isOpen = true, setIsOpen }: ElementProps) => {
   const isWaitingForOrderToIndex = useAppSelector(getHasUncommittedOrders);
   const showCurrentMarket = isTablet || view === PanelView.CurrentMarket;
 
+  const numTotalPositions = (useAppSelector(selectParentSubaccountOpenPositions) ?? EMPTY_ARR)
+    .length;
```

Comment:
ok for now. but i feel like we get this num in multiple places. might be good to move to a hook